tools/coqdep/lib/fl.ml

```diff
@@ -86,18 +86,54 @@ let rec find_plugin meta_file plugin_name path p { Fl_metascanner.pkg_defs ; pkg
     let path = path @ [find_plugin_field "directory" "." c.Fl_metascanner.pkg_defs] in
     find_plugin meta_file plugin_name path ps c
 
+let to_relative_path : string -> string = fun full_path ->
```

Comment:
A comment here wouldn't hurt IMVHO.

Reply:
I added documentation for both file functions, and also ran some tests locally:
```
Tests for [normalize_tests]:
 - normalize "." = "."
 - normalize "./dir1/dir2/test.v" = "dir1/dir2/test.v"
 - normalize "dir1/dir2/" = "dir1/dir2/"
 - normalize "/" = "/"
 - normalize "/tmp/dir" = "/tmp/dir"
 - normalize "/tmp/dir/" = "/tmp/dir/"
 - normalize "/tmp/dir/../dir2/../dir1/././dir3" = "/tmp/dir1/dir3"
 - normalize "/tmp/dir/./" = "/tmp/dir/"
 - normalize "/tmp/dir/.//" = "/tmp/dir//"
Tests for [to_relative_path] (PWD = "/home/rodolphe/dev/coq/coq-coqdep"):
 - to_relative_path "/tmp/" = "../../../../../tmp/"
 - to_relative_path "/home/rodolphe/" = "../../../"
 - to_relative_path "/home/rodolphe/../john" = "../../../../john"
 - to_relative_path "/home/rodolphe/../../tmp/../tmp/file.txt" = "../../../../../tmp/../tmp/file.txt"
```
This helped uncover a bug, but we would never have hit it here, since we are working with path to actual files (not directories).